application.py

```diff
@@ -0,0 +1,207 @@
+import csv
+import datetime
+import re
+
+import matplotlib.pyplot as plt
+
+"""
+    Initialized categories list to choose correct option for add new spending.
+    todo:: add some more categories, create custom categories
+    todo:: (add them to other file and read to list to manipulate with them!)
+"""
+
+categoriesList = ["Food", "Clothes", "Entertainment", "Transportation", "Other"]
```

Comment:
let's add functionality to add our own categories

Reply:
correct me if it is presented here

Reply:
already created and will be represented in future commit